Admin/FaqAdmin.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Grossum\FaqBundle\Admin;
+
+use Sonata\AdminBundle\Admin\Admin;
+use Sonata\AdminBundle\Datagrid\DatagridMapper;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type as FormType;
+use Grossum\FaqBundle\Entity\EntityManager\FaqManager;
```

Comment:
Usually we group namespase by "priority". At first, it is symfony bundels then doctrine then other and at the end your bundle namespases. All this groups we separate with new line

`
use Symfony\Component\Form\Extension\Core\Type as FormType;

use Sonata\AdminBundle\Admin\Admin;
use Sonata\AdminBundle\Datagrid\DatagridMapper;
use Sonata\AdminBundle\Datagrid\ListMapper;
use Sonata\AdminBundle\Form\FormMapper;

use Grossum\FaqBundle\Entity\EntityManager\FaqManager;`